src/fmu/dataio/dataio.py

```diff
@@ -969,18 +970,25 @@ def export(
             ).export(obj)
 
         logger.info("Object type is: %s", type(obj))
+        self._update_check_settings(kwargs)
```

Comment:
This `_update_check_settings` is called inside the `generate_metadata` also, but this PR removes the call to the `generate_metadata` function and hence we should call this function again here (until we remove it in fmu-dataio 3.0.0).